src/subscription/Subscription.js

```diff
@@ -79,19 +79,44 @@ Subscription.prototype.subscribed = function() {
  * @return {boolean}
  */
 Subscription.prototype.alive = function() {
-    return this.subscribed() && Date.now() < this.expirationTime();
+    return this.subscribed() && Date.now() < (this.expirationTime() - this._renewHandicapMs);
```

Comment:
`- this._renewHandicapMs` should be in `expirationTime()` function in order to keep the previous logic.